### DISCUSSION TITLE
Re-enable auto auth?

### DISCUSSION THREAD

**Discussion by jerdrb** [homebridge-simplisafe3/homebridge-simplisafe3#381] [Feature Requests]
Seems like with the reCaptcha gone for good, could you re-enable the auto authorization similar to python simplisafe?

**Comment by shamoon** [comment]
Unfortunately it's not the recaptcha, they seem to have completely changed the auth process in a way that I have not been able to circumvent. To be honest it was always a matter of time without their official support (which they have never responded to)

**Reply by jerdrb** [reply]
Ah, got it. Yes, I have noticed that SS may be falling into the trap that certain tech companies get into which is, focus on selling new HW, not making the ones you have already have better features...

Thanks for the quick reply at least!

**Reply by shamoon** [reply]
100% that, has been for a long time TBH. Ages ago for example they promised (official) HK support and then bailed on that promise. Will be interested to see what happens with Matter, eventually. Its a shame, working with third-parties brings obvious value to their product but they don't seem interested

**Comment by thetaz001** [comment]
So am i reading this correct, the Homebridge link possibility to SS is no longer possible.
i am having the same issue i can no longer get SS to Auth my plug in no matter what i do..
i guess this is a dead topic since SS sucks and doesnt support the HK technology :(?

**Reply by shamoon** [reply]
No, it can still be done, just not automagically. Please read the docs.

**Comment by thetaz001** [comment]
Thanks so much!
Apologies I didn’t realize you updated the docs!!
That worked!
you guys rock for keeping this working!!
thank you
Fred
